docs/content/templates/feature-guide-template.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Feature name
+metaTitle: Feature name - JavaScript Data Grid | Handsontable
+description: SEO description. Max 160 characters. Start with an action verb.
+permalink: /feature-name
+canonicalUrl: /feature-name
+tags:
+  - additional
+  - search
+  - tags
+react:
+  metaTitle: Feature name - React Data Grid | Handsontable
+searchCategory: Guides
```

Comment:
```suggestion
---
id: Unique page id for default vanilla JS version (e.g 45dody2p)
title: Feature name
metaTitle: Feature name - JavaScript Data Grid | Handsontable
description: SEO description. Max 160 characters. Start with an action verb.
permalink: /feature-name
canonicalUrl: /feature-name
tags:
  - additional
  - search
  - tags
react:
  id: Unique page id for React version (e.g f8xulniq)
  metaTitle: Feature name - React Data Grid | Handsontable
searchCategory: Guides
```
I'd update the template with the page ID that will be required after merging: https://github.com/handsontable/handsontable/pull/10163.

Reply:
Thanks! Added: [dacdb07](https://github.com/handsontable/handsontable/pull/10176/commits/dacdb076eb53949a352afc82e4655cb02ce8efa8)